mlflow/tracking/client.py

```diff
@@ -581,12 +581,16 @@ def create_experiment(
 
         .. code-block:: python
             :caption: Example
-
+            from pathlib import Path
```

Comment:
```suggestion

            from pathlib import Path
```

A new line needs to inserted.

Reply:
I believe it does not cause any issue in rendering but more in the spirit of uniformity ?

Reply:
This breaks the documentation build:

https://app.circleci.com/pipelines/github/mlflow/mlflow/13128/workflows/ea7493a1-125c-4eb2-b305-edb9589c165b/jobs/23129

Reply:
```
/home/circleci/project/mlflow/tracking/client.py:docstring of mlflow.tracking.client.MlflowClient.create_experiment:11: WARNING: Error in "code-block" directive:
invalid option block.

.. code-block:: python
    :caption: Example
    from pathlib import Path
    from mlflow import MlflowClient

    # Create an experiment with a name that is unique and case sensitive.
    client = MlflowClient()
    experiment_id = client.create_experiment(
        "Social NLP Experiments",
        artifact_location=Path.cwd().joinpath("mlruns").as_uri(),
        tags={"version": "v1", "priority": "P1"},
    )
    client.set_experiment_tag(experiment_id, "nlp.framework", "Spark NLP")

    # Fetch experiment metadata information
    experiment = client.get_experiment(experiment_id)
    print("Name: {}".format(experiment.name))
    print("Experiment_id: {}".format(experiment.experiment_id))
    print("Artifact Location: {}".format(experiment.artifact_location))
    print("Tags: {}".format(experiment.tags))
    print("Lifecycle_stage: {}".format(experiment.lifecycle_stage))

```

Reply:
I ran `make livehtml` and was able to see docs update. Nevertheless updated the code with newline. Will monitor docs ci job going forward. Thanks !